tools/hidl/hidlcomp.cpp

```diff
@@ -5371,6 +5384,9 @@ void EspServInfo::write_esp_binding()
     }
     outs("\tDBGLOG(\"Client version: %g\", context.getClientVersion());\n");
 
+    // getXSDDefinition(StringBuffer &content, const char *packageName, const char *serviceName, double version, const char *method);
```

Comment:
Is this the commented out code you said you were removing?

Reply:
This is the original generated code that calls the hardcoded code in the service. I left if for reference until it fully removed.

Reply:
Still think this should just be removed, but I assume this will be removed as part of one of your next PRs.  Can be merged as is.